HousingRepairsOnlineApi/Helpers/RepairDescriptionExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using HousingRepairsOnlineApi.Domain;
+
+namespace HousingRepairsOnlineApi.Helpers;
+
+public static class RepairExtensions
+{
+    public static string AppendLocationDescription(this RepairDescription repairDescription)
```

Comment:
The `Append` prefix suggests LocationDescription is being append to something but the name doesn't specify what that is, which would make difficult to understand later.

I would suggest renaming to `CombinedDescriptionTexts` or something equivalent:
```suggestion
    public static string CombinedDescriptionTexts(this RepairDescription repairDescription)
```